test/src/main/java/org/apache/accumulo/test/zookeeper/ZooKeeperTestingServer.java

```diff
@@ -47,16 +46,14 @@
    * Instantiate a running zookeeper server - this call will block until the server is ready for
    * client connections. It will try three times, with a 5 second pause to connect.
    */
-  public ZooKeeperTestingServer() {
-    this(PortUtils.getRandomFreePort());
+  public ZooKeeperTestingServer(File tmpDir) {
+    this(tmpDir, PortUtils.getRandomFreePort());
   }
 
-  private ZooKeeperTestingServer(int port) {
+  private ZooKeeperTestingServer(File tmpDir, int port) {
 
```

Comment:
I wonder if adding a check here that tmpDir is indeed a directory would be good. Not sure if its needed but we could add something like:
```
Preconditions.checkArgument(tmpDir.isDirectory());
```
or
```
Preconditions.checkArgument(tmpDir.isDirectory() || tmpDir.mkdir());
```